runtime/ftplugin/javascript.vim

```diff
@@ -50,8 +50,10 @@ let s:bin_dir = finddir('node_modules/.bin', '.;')
 if !empty(s:bin_dir)
     let s:bin_dir = fnamemodify(s:bin_dir, ':p')
 
+    let path_separator = (has('win64') || has('win32') || has('win16')) ? ';' : ':'
```

Comment:
```suggestion
    let path_separator = has('win32') ? ';' : ':'
```
When `has('win64')` is true, `has('win32')` is also true.
The win16 support was dropped a few years ago.

Reply:
Thank you. Done.